source/core/Date.cpp

```diff
@@ -176,23 +176,19 @@ namespace Vireo {
             seconds += kSecondsPerDay * dayOfMonth[i];
             if (seconds > secondsOfYear) {
                 currentMonth = i;
-                secondsOfMonth = secondsOfYear - secondsOfMonth;
+                secondsOfMonth = Int32(secondsOfYear - secondsOfMonth);
                 break;
             }
         }
 
         // Get timezone abbrevation
         char timeZoneAbbr[kTempCStringLength] = "TODO-TimeZone";
-#if (kVireoOS_linuxU || kVireoOS_macosxU)
-        time_t rawtime;
-        struct tm* timeinfo;
-        time(&rawtime);
-        timeinfo = localtime(&rawtime);
-        snprintf(timeZoneAbbr, sizeof(timeZoneAbbr), "%s", timeinfo->tm_zone);
-#elif kVireoOS_emscripten
-        TempStackCString result;
-        result.AppendCStr(jsTimestampGetTimeZoneAbbr());
-        snprintf(timeZoneAbbr, sizeof(timeZoneAbbr), "%s", result.BeginCStr());
+#if (kVireoOS_linuxU || kVireoOS_macosxU || kVireoOS_emscripten)
+        time_t rawtime = timestamp.Integer() - kStdDT1970re1904;
+        struct tm timeinfo;
+        localtime_r(&rawtime, &timeinfo);
+        snprintf(timeZoneAbbr, sizeof(timeZoneAbbr), "%s", timeinfo.tm_zone);
+// #elif kVireoOS_emscripten variant deleted; localtime_r works correctly
```

Comment:
Remove the comment, in the future is just going to cause confusion about what variant was deleted.

It was helpful for the code review.